src/resources.js

```diff
@@ -19,7 +19,7 @@ let $Info       = Vue.resource(origin + '/api/0/info');
 let $Bucket     = Vue.resource(origin + '/api/0/buckets/{id}?force=1');
 let $Event      = Vue.resource(origin + '/api/0/buckets/{id}/events');
 let $EventCount = Vue.resource(origin + '/api/0/buckets/{id}/events/count');
-let $Query      = Vue.resource(origin + '/api/0/query/?name={name}&start={start}&end={end}&cache={cache}');
```

Comment:
Would it be much work to keep the old way of doing things in addition to the new one? (being able to specify period to query by URL parameters if you have only one period to query)

Reply:
Not really, I just think it would be unnecessary to have two ways to do one thing without any significant benefit.

Reply:
There is some benefit when it comes to the OpenAPI/Swagger spec, and therefore the API Browser. If it's just a couple lines I'd appreciate if you readded support for it in aw-server :+1:

Reply:
> There is some benefit when it comes to the OpenAPI/Swagger spec, and therefore the API Browser. 

Well, the "timeperiods" field is still marshalled so we still don't break the OpenAPI/Swagger spec in any way as far as i understand. It just doesn't make sense to me to have two options in an API which depend or whether you want to provide one or more of a data type. An API should be rather straight forward just by looking at the definition, you shouldn't have to explain it with too much text. In this case we would have to document both options, document which one is prefered over the other in different situations, document which one is considered first priority if both are specified etc. which just screams bad design to me.

Reply:
Fair enough.